### PR TITLE
Resolves #183 : add endpoints for moving and copying vApps

### LINKED ISSUE #183: Add vApp copy/move endpoint methods

### PR DESCRIPTION
**Pull request by bsnyder788**

### CODE REVIEW

File: src/sdk/model/vapp/vapp-copy-move-request.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   */
+  get description(): string {
+    return this._json.description;
+  }
+
+  /**
+   * Get vm specs.
+   * @returns {Array<VappChildVmCopyMoveRequestJson>}
+   */
+  get vmSpecs(): Array<VappChildVmCopyMoveRequestJson> {
```
Comment (cfsnyder) [inline src/sdk/model/vapp/vapp-copy-move-request.ts]: should return an instance of the wrapper class, not the JSON interface

Comment (bsnyder788) [reply src/sdk/model/vapp/vapp-copy-move-request.ts]: @cfsnyder fixed

File: src/sdk/model/vapp/vapp-copy-move-request.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  get description(): string {
+    return this._json.description;
+  }
+
+  /**
+   * Get vm specs.
+   * @returns {Array<VappChildVmCopyMoveRequest>}
+   */
+  get vmSpecs(): Array<VappChildVmCopyMoveRequest> {
+    return this.vmSpecs;
```
Comment (cfsnyder) [inline src/sdk/model/vapp/vapp-copy-move-request.ts]: stack overflow

Comment (bsnyder788) [reply src/sdk/model/vapp/vapp-copy-move-request.ts]: fixed

**Pull request merged**